src/utils/collapseSourcemaps.ts

```diff
@@ -174,34 +168,55 @@ export default function collapseSourcemaps(
 			},
 			[source]
 		);
+	};
+}
+
+function unionSourcemaps(
```

Comment:
I would also prefer if this function name could start with a `verb`. Is maybe `collapseSourcemap` (singular) what is happening here? Or `getCollapsedSourcemap`?